### DISCUSSION TITLE
Request for assistance regarding the extension license

### DISCUSSION THREAD

**Discussion by axelpichon** [aclap-dev/video-downloadhelper#1586] [Help and Q&A]
### Your question

I purchased a license for a Chrome extension, but it is not compatible with YouTube, which was the primary reason for my purchase. I was advised to use Firefox instead, but I am being asked to pay for another license. Could you please assist me?

### Addon & CoApp version and details

version: 9.1.0.44
target: mozilla
channel: stable
lang: fr
coapp: {"found":true,"path":"/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp","version":"2.0.19","new_version":false}
license: {"mismatch":true,"key":"7457ab6738354b72","other_browser":"Chrome","this_browser":"Firefox"}
platform: aarch64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:131.0) Gecko/20100101 Firefox/131.0
{
    "used_history_button": true,
    "download_directory": "/Users/axelpichon/Downloads",
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": false,
    "open_count_store": 6,
    "successfull_dl": 3
}

**Comment by jcfrog** [comment]
hi,
please send us a message at licenses@downloadhelper.net
